### ISSUE TITLE
Font menu isn't correctly labeled.

### ISSUE THREAD

**Issue by rkingett** [writeas/writefreely#365]
While running a screen reader, using either, the Classic editor or the modern editor, the font menu isn't labeled correctly to screen readers. NVDA, a free screen reader, just says, Graphic, new. Link.

After the menu is opened, a keyboard user cannot change items using the keyboard when the screen reader, NVDA, is passing keys through directly through the browser.

### Steps to reproduce (if necessary)
1. Go to create a new post. Run NVDA, a free screen reader. '...'
2. Press enter on the first, NEW, graphic label below the text area. 

### Expected behavior
I was expecting these options would be standard combo box or radio buttons or checkboxes. I didn't expect there to be a link menu structure here.

I am using this on Write.as. Not my own instance. Standard radio buttons or checkboxes or combo boxes would be better for accessibility.

**Comment by rkingett** [comment]
To also add to this, the sub menus in this menu are extremely poorly labeled, and do not maintain keyboard focus very well. It would be better if we could nest these options in a standard box on the page, with standard radio buttons, checkboxes, and HTML 5 dropdown menus.

**Comment by rkingett** [comment]
I'd like to suggest we incorporate [this accessibility bot](https://github.com/marketplace/accesslint) but it is not a one stop accessibility tool. Issues would still need to be watched and corrected manually.

If this is being used in educational settings, the menus on the post page, font menu, and otherwise, need to be corrected because this could be a great tool for teachers working with Disabled students.

@leonstafford would this bot be good, or would you recommend another bot?

**Comment by leonstafford** [comment]
@rkingett, hmm, don't know enough about AccessLint to recommend or not. Being free for personal GitHub accounts, I should be trying it, now that I've brought all my stuff in from an Org to Personal.

As WriteAs charges for their service, they should be fine to pay for AccessLint if it will catch things like this.

@thebaer - does it look like AccessLint would catch issues like this?

**Comment by leonstafford** [comment]
@rkingett, worked well in [this simple case](https://github.com/leonstafford/ljsdotdev/pull/2)... not yet sure about in generated HTML code, which would be more useful for all the static site generation tools, like Hugo and I assume WriteAs.

My guess is that using project-specific CLI tools will make more sense, as AccessLint can't be expected to magically understand all project types and their specific way of generating output.
